### PR TITLE
Modify pay portion

### PR DESCRIPTION
**Pull request by hensha256**

### CODE REVIEW

File: contracts/modules/Payments.sol [hunk 1]
```diff
@@ -45,12 +45,16 @@ abstract contract Payments is RouterImmutables {
         if (token == Constants.ETH) {
             uint256 balance = address(this).balance;
             uint256 amount = (balance * bips) / FEE_BIPS_BASE;
-            recipient.safeTransferETH(amount);
+            uint256 fixedAmount = (amount * PAYMENT_AMOUNT_BIPS) / FEE_BIPS_BASE;
```
Comment (snreynolds) [inline contracts/modules/Payments.sol:48]: do we want to handle case when this these are unset and `fixedAmount` is 0? I.e if contract is initialized with `PAYMENT_AMOUNT_BIPS` is 0 and/or `PAYMENT_RECIPIENT` is addr 0? Would save a call

Comment (hensha256) [reply contracts/modules/Payments.sol:48]: I think in that case we just wouldnt deploy with this feature. it wont be modifiable so if we want `PAYMENT_AMOUNT_BIPS==0` we would remove this code entirely

**Review by marktoda — APPROVED** [review]

File: contracts/modules/Payments.sol [hunk 2]
```diff
@@ -61,12 +61,16 @@ abstract contract Payments is RouterImmutables {
...
```
Comment (ewilz) [inline contracts/modules/Payments.sol:64]: Our past routers have had the option to let third parties use this feature too, are we ok with getting rid of that?

Comment (hensha256) [reply contracts/modules/Payments.sol:64]: I'm confused... what about this doesnt let a third party use it?
The design is:
- x% is taken from the user
- `PAYMENT_AMOUNT_BIPS`% of the x% goes to the recipient
- the rest of the x% goes to the address specified by the 3rd party

**Pull request merged**